server/rpc_server.go

```diff
@@ -303,14 +293,12 @@ func (s *rpcServer) ServeConn(sock transport.Socket) {
 			// try get a new codec
 			if cf, err = s.newCodec(ct); err != nil {
 				// no codec found so send back an error
-				if err := sock.Send(&transport.Message{
+				sock.Send(&transport.Message{
```

Comment:
why error not checked?

Reply:
> why error not checked?

Error check previous just set `gerr` for global error tracking, unnecessary, now.

Reply:
yes, but for now - if sock.Send returns error why we not log it and not use it ?

Reply:
if it not needed for client i think that we need to log it via logger.Errorf